src/components/pages/AdminHome/CourseCard.js

```diff
@@ -3,13 +3,13 @@ import { useHistory } from 'react-router-dom';
 import { Card } from 'antd';
 import { connect, useDispatch } from 'react-redux';
 import { getCourses } from '../../../redux/actions/coursesActions';
-import { useOktaAuth } from '@okta/okta-react';
 import '../../../styles/AdminDashboardHome/index.less';
+//TO-DO: Implement Auth0
 
 const CourseCard = props => {
   const history = useHistory();
   const dispatch = useDispatch();
-  const idToken = useOktaAuth().oktaAuth.getIdToken();
+  const idToken = 'To be stubbed out and replaced with auth0';
```

Comment:
Another false value that'll break `axiosWithAuth` or most anything else that tries to use it.